### PR TITLE
create add task function

### PR DESCRIPTION
**Pull request by huutuandn96**
- Create add task function
- Return current task view
- Validate add task form
- Show errors validate in view

### CODE REVIEW

**Comment by huutuandn96** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -19,3 +19,4 @@
 });
 
 Route::get('/task', [TaskController::class, 'index'])->name('task.index');
+Route::post('/task', [TaskController::class, 'store'])->name('task.store');
```
Comment (quoctd-0607) [inline routes/web.php]: dùng resource with only cho gọn route nhé

**Comment by huutuandn96** [comment]
ready

**Comment by quoctd-0607** [comment]
merged

**Pull request merged**